test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SqlProjects/SqlProjectsServiceTests.cs

```diff
@@ -208,77 +208,77 @@ public async Task TestNoneScriptOperations()
             Assert.IsTrue(File.Exists(absolutePath), $"{absolutePath} expected to still exist on disk");
 
             // Re-add to set up for Delete
             requestMock = new();
-            await service.HandleAddNoneScriptRequest(new SqlProjectScriptParams()
+            await service.HandleAddNoneItemRequest(new SqlProjectScriptParams()
             {
                 ProjectUri = projectUri,
                 Path = relativePath
             }, requestMock.Object);
 
-            requestMock.AssertSuccess(nameof(service.HandleAddNoneScriptRequest));
-            Assert.AreEqual(1, service.Projects[projectUri].NoneScripts.Count, "NoneScripts count after re-add");
+            requestMock.AssertSuccess(nameof(service.HandleAddNoneItemRequest));
+            Assert.AreEqual(1, service.Projects[projectUri].NoneScripts.Count, "NoneItems count after re-add");
 
             // Validate moving a None script
             string movedScriptRelativePath = @"SubPath\RenamedNoneIncludeFile.json";
             string movedScriptAbsolutePath = Path.Join(Path.GetDirectoryName(projectUri), FileUtils.NormalizePath(movedScriptRelativePath));
             Directory.CreateDirectory(Path.GetDirectoryName(movedScriptAbsolutePath)!);
 
             requestMock = new();
-            await service.HandleMoveNoneScriptRequest(new MoveItemParams()
+            await service.HandleMoveNoneItemRequest(new MoveItemParams()
             {
                 ProjectUri = projectUri,
                 Path = relativePath,
                 DestinationPath = movedScriptRelativePath
             }, requestMock.Object);
 
-            requestMock.AssertSuccess(nameof(service.HandleMoveNoneScriptRequest));
+            requestMock.AssertSuccess(nameof(service.HandleMoveNoneItemRequest));
             Assert.IsTrue(File.Exists(movedScriptAbsolutePath), "Script should exist at new location");
-            Assert.AreEqual(1, service.Projects[projectUri].NoneScripts.Count, "NoneScripts count after move");
+            Assert.AreEqual(1, service.Projects[projectUri].NoneScripts.Count, "NoneItems count after move");
 
             // Validate deleting a None script
             requestMock = new();
-            await service.HandleDeleteNoneScriptRequest(new SqlProjectScriptParams()
+            await service.HandleDeleteNoneItemRequest(new SqlProjectScriptParams()
             {
                 ProjectUri = projectUri,
                 Path = movedScriptRelativePath
             }, requestMock.Object);
 
-            requestMock.AssertSuccess(nameof(service.HandleDeleteNoneScriptRequest));
-            Assert.AreEqual(0, service.Projects[projectUri].NoneScripts.Count, "NoneScripts count after delete");
+            requestMock.AssertSuccess(nameof(service.HandleDeleteNoneItemRequest));
+            Assert.AreEqual(0, service.Projects[projectUri].NoneScripts.Count, "NoneItems count after delete");
```

Comment:
that's coming from another library, which has the updates in-flight.  Since they don't need to be linked and the higher-stack changes are more important, I figured I'd do the updates as they're ready